pkgs/tools/misc/bandwidth/default.nix

```diff
@@ -19,7 +19,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ nasm ];
 
-  buildFlags = [ arch ];
+  buildFlags = [ arch ]
+    ++ stdenv.lib.optionals stdenv.cc.isClang [ "CC=clang" "LD=clang" ];
```

Comment:
cc-wrapper does set `CC` (https://github.com/NixOS/nixpkgs/blob/67fe5d46d4cd1c5383462bc136cd3c4ccc8cefc2/pkgs/build-support/cc-wrapper/setup-hook.sh#L42-L43).

I think the real problem is that the Makefile isn't inheriting the value of `CC` from the environment, so we probably need to make `CC=$CC` a default flag.